CODE/Sources/PennyAPI/Sponsors/SponsorLambda.swift

```diff
@@ -183,7 +197,7 @@ struct AddSponsorHandler: LambdaHandler {
         
         // Throw if adding new role response is invalid
         guard 200...299 ~= removeRoleResponse.status.code else {
-            context.logger.error("Failed to remove \(role.rawValue) role from user \(userDiscordID) with error: \(removeRoleResponse.status.description)")
+            context.logger.error("Failed to remove \(role.rawValue) role from user \(userDiscordID) with error: \(removeRoleResponse.status.description) and body: \(removeRoleResponse.body?.debugDescription ?? "empty")")
```

Comment:
This is a `ByteBuffer` right? We should convert it to a string properly as I don't think description will actually print the body